apps/server/src/config/manifest.ts

```diff
@@ -0,0 +1,3 @@
+import pkg from '../../package.json' assert { type: 'json' };
+
+export const { name, version } = pkg;
```

Comment:
Does this work in a single statement?

```suggestion
export { name, version } from '../../package.json' assert { type: 'json' };
```

Reply:
good question.
while that would look much simpler, I like the idea that we can do validations or transformations in this file in the future.

Reply:
it looks simpler using a single line, however it doesn't work it throws the error `The requested module '../../package.json' does not provide an export named 'name'`.